libraries/wallet/include/graphene/wallet/wallet.hpp

```diff
@@ -1538,6 +1538,10 @@ class wallet_api
          
       order_book get_order_book( const string& base, const string& quote, unsigned limit = 50);
 
+      signed_transaction multisig_sign_transaction( signed_transaction tx,
+                                                    const vector<string> &wif_keys,
```

Comment:
This is a wallet API call. I would have expected that the wallet adds required signatures for the keys it knows, not that you have to pass private keys into the call.